description.md

```diff
@@ -38,6 +38,71 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Assignments
 
+Assignments are practical activities that learners undertake individually or in teams over the span of multiple weeks.
```

Comment:
```suggestion
Assignments are practical activities that learners undertake individually or in teams over o longer timespan.
```

Reply:
I would also use the wording "unassisted". Assistance can be provided during office hours or online, but learners do things by themselves.